src/main/java/ua/danit/final_project/configuration/security/JwtAuthenticationRequest.java

```diff
@@ -0,0 +1,16 @@
+package ua.danit.final_project.configuration.security;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+
+@Data
+@AllArgsConstructor
+public class JwtAuthenticationRequest {
+
+  private String userName;
+  private String userPassword;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "userPassword" private field. ([squid:S1068](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=173&issues=AWb1IgB6W3Y69bz5KWFy&open=AWb1IgB6W3Y69bz5KWFy)

[ISSUE_KEY]: AWb1IgB6W3Y69bz5KWFy